src/test/java/io/javalin/openapi/TestOpenApi.kt

```diff
@@ -658,4 +658,27 @@ class TestOpenApi {
         assertThat(userJson.formatJson()).isEqualTo(userJsonExpected.formatJson())
         assertThat(userWithIdJson.formatJson()).isEqualTo(userWithIdJsonExpected.formatJson())
     }
+
+    @Test
+    fun `createSchema() work with security requirements`() {
```

Comment:
I had to remove all the parens `()` from test names because it was causing issue with the test runner in IntelliJ.

`reateSchema() work with security requirements` -> `reateSchema works with security requirements`